### PR TITLE
Kodarie working

### PR DESCRIPTION
**Pull request by kodariewhs**
merging with the master

### CODE REVIEW

File: app/src/main/java/uk/ucl/group21/currentviewapp/Questionnaire_Overview.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+public class Questionnaire_Overview extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_questionnaire__overview);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
```
Comment (tvaupel) [inline app/src/main/java/uk/ucl/group21/currentviewapp/Questionnaire_Overview.java]: This is useless lol

File: app/src/main/res/layout/activity_questionnaire__overview.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="?attr/actionBarSize"
+            android:background="?attr/colorPrimary"
+            app:popupTheme="@style/AppTheme.PopupOverlay" />
+
+    </android.support.design.widget.AppBarLayout>
+
+    <include layout="@layout/content_questionnaire__overview" />
+
+    <android.support.design.widget.FloatingActionButton
```
Comment (tvaupel) [inline app/src/main/res/layout/activity_questionnaire__overview.xml]: delete this!
lines 26 till 32.

File: app/src/main/res/menu/menu_main_activities.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="com.example.whsgemac.databasesqlitetrail.MainActivities">
+    <item
+        android:id="@+id/action_settings"
+        android:orderInCategory="100"
+        android:title="action_settings"
```
Comment (tvaupel) [inline app/src/main/res/menu/menu_main_activities.xml]: title is wrong!
not a reference to strings.xml

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -2,4 +2,5 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
```
Comment (tvaupel) [inline app/src/main/res/values/dimens.xml]: Not needed anymore!

**Comment by kodariewhs** [comment]
all questionnaire related screen done  and have solved the conflicts (originally there are some minor conflicts between our changes)

File: app/src/main/res/values-v21/styles.xml [hunk 5]
```diff
@@ -1,8 +1,23 @@
 <resources>
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
```
Comment (tvaupel) [inline app/src/main/res/values-v21/styles.xml]: resolve merge conflicts.

**Comment by tvaupel** [comment]
closing pull request. New one has already been submitted & changes required

**Pull request closed without merging**